frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.tsx

```diff
@@ -27,9 +27,18 @@ const AdminShoutouts: React.FC = () => {
                 </Item.Header>
                 <Item.Meta>
                   From:{' '}
-                  {shoutout.giver
-                    ? `${shoutout.giver.firstName} ${shoutout.giver.lastName}`
-                    : '(Former member)'}
+                  {(() => {
+                    switch (shoutout.isAnon) {
+                      case true:
+                        return 'Anonymous';
+                      case false:
+                        return shoutout.giver
+                          ? `${shoutout.giver.firstName} ${shoutout.giver.lastName}`
+                          : '(Former member)';
+                      default:
+                        return 'Anonymous';
```

Comment:
Someone else can come in on this if it was discussed offline already, but I didn't see anything in the notion issue that said we were going to anonymize shoutouts from the admin view. Is that new functionality that we've agreed upon? @JacksonStaniec 

If it is intended...

This can be rewritten shorter with an if-statement since you only have two results, non-anonymous and anonymous.

I think it makes more sense to handle this logic similarly in `ShoutoutCard.tsx` if you changed it here. Take a look at how Anonymous users were previously handled in lines 17-37 in `ShoutoutCard.tsx` and see if you can match them.

Reply:
Yea this was discussed offline. This is the page that is going to be shown during all-hands, so we need to make sure that people are anonymous if they want to be.

Reply:
@andrew032011 I made the logic a bit shorter. Are you also proposing changing the admin view to a `List` of type `Card` instead of type `Item` to match the type in the member view?

Reply:
I think what you have now is fine. More or less I was referring to what Henry mentioned.